protobuf_files/README_UPDATE_PROTOBUF_FILES.md

```diff
@@ -22,3 +22,22 @@ Updating the compiled protobuf files is just as easy as the following steps:
 
 * <https://github.com/steamdatabase/protobufs>
 * <https://github.com/ValvePython/steam>
+
+## Installing and generating protobufs on Mac
+
+First time Install Python OpenSSL Certificates so the request library can pull Stream protobufs.
+
+1. Install python 3.7 from https://www.python.org/downloads/. It puts Python in your Applications directory
+2. `"/Applications/Python 3.7/Install Certificates.command"`  You must install for Python 3.7.
```

Comment:
hopefully i can just install `certifi` and it'll work. This is done in https://github.com/ABaumher/galaxy-integration-steam/blob/MacOS_Fix/requirements/install.txt#L13 and https://github.com/ABaumher/galaxy-integration-steam/blob/MacOS_Fix/requirements/app.txt#L7